pkgs/by-name/cl/clipse/package.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub 
+}:
+
+buildGoModule rec {
+  pname = "clipse";
+  version = "0.0.6";
+
+  src = fetchFromGitHub {
+    owner = "savedra1";
+    repo = "clipse";
+    rev = "v${version}";
+    hash = "sha256-zCsSfnZbhOYe1ysnGFO6VYZKVYCNZY0DvP2FB3EXFDM=";
+  };
+
+  vendorHash = "sha256-GIUEx4h3xvLySjBAQKajby2cdH8ioHkv8aPskHN0V+w=";
+
+  meta = with lib; {
+    description = "A useful clipboard manager TUI for Unix";
+    homepage = "https://github.com/savedra1/clipse";
+    license = licenses.gpl3Only;
```

According to the upstream repo's [`LICENCE`](https://github.com/savedra1/clipse/blob/main/LICENSE) the code is licensed under MIT.
```suggestion
    license = licenses.mit;
```

---

Thanks again! this is now updated also.